### PR TITLE
Rename jobs so they're not all named "build" or "analyze"

### PR DESCRIPTION
**Pull request by tnleeuw**

### CODE REVIEW

**Comment by philipsens** [comment]
Wouldn't it be nice to have a name attribute for the build job as well?

**Comment by philipsens** [comment]
> > Wouldn't it be nice to have a name attribute for the build job as well?
> 
> For which build job? 😉
> 
> I'm not sure it would really add much value though to be honest.

We are changing the job keys, so the display name also changes and causes less confusion, right? So in that scenario, wouldn't it be nice to add a name attribute so we can use capital letters and spaces? That the job keys don't matter as much anymore.

**Comment by philipsens** [comment]
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idname

File: .github/workflows/angular-karma-tests.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ on:
       - 'release/[0-9]+.[0-9]+'
 
 jobs:
-  build:
+  run-karma-tests:
```
Comment (philipsens) [inline .github/workflows/angular-karma-tests.yml:19]:
```suggestion
  run-karma-tests:
    name: Run Karma tests
```

File: .github/workflows/compare-frankdoc.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ on:
         required: true
 
 jobs:
-  build:
+  compare-frank-doc:
```
Comment (philipsens) [inline .github/workflows/compare-frankdoc.yml:13]:
```suggestion
  compare-frank-doc:
    name: Compare Frank!Doc
```

File: .github/workflows/maven-build.yml [hunk 3]
```diff
@@ -40,7 +40,7 @@ on:
       - "docker/**"
 
 jobs:
-  build:
+  build-maven-artefacts:
```
Comment (philipsens) [inline .github/workflows/maven-build.yml:43]:
```suggestion
  build-maven-artifacts:
    name: Build Maven artifact
```

**Comment by tnleeuw** [comment]
Github is configured that a job named "build" is required, but there's no more such job. So the commit-checks never complete.
Where should this be configured?

**Review by philipsens — APPROVED** [review]
Thank youu!

**Pull request merged**